modules/nw-sriov-dpdk-example-mellanox.adoc

```diff
@@ -6,47 +6,48 @@
 [id="example-vf-use-in-dpdk-mode-mellanox_{context}"]
 = Using a virtual function in DPDK mode with a Mellanox NIC
 
+You can create a network node policy and create a Data Plane Development Kit (DPDK) pod using a virtual function in DPDK mode with a Mellanox NIC.
 .Prerequisites
```

Comment:
This line is rendering as ".Prerequisites" in the preview. I think you might need a new line before this line so that it is rendered as a dot heading.